tests/vertical_playground.html

```diff
@@ -396,6 +399,54 @@
           </value>
         </block>
       </category>
+      <category name="Pen">
+        <block type="pen_clear"></block>
+        <block type="pen_stamp"></block>
+        <block type="pen_penup"></block>
+        <block type="pen_pendown"></block>
```

Comment:
Not important, but we put "pen down" before "pen up" since you usually have to use that one first :)